app/templates/course/introduction.hbs

```diff
@@ -4,6 +4,8 @@
       @featureSuggestion={{this.authenticator.currentUser.pendingProductWalkthroughFeatureSuggestion}}
       class="mb-6"
     />
+  {{else if (not this.authenticator.isAuthenticated)}}
```

Comment:
This isn't required - we don't expose this page to unauthenticated users (at the moment at least)